default.yaml

```diff
@@ -152,6 +152,7 @@ onChainWallet:
 
 apollo:
   playground: true
+  playgroundUrl: "https://api.staging.galoy.io/graphql"
```

Comment:
@bodymindarts can I make the default here http://localhost:4002/graphql ?

It's annoying for local dev to have the default on staging.